lib/internal/Magento/Framework/App/FrontController.php

```diff
@@ -137,6 +137,9 @@ public function dispatch(RequestInterface $request)
         $this->validatedRequest = false;
         $routingCycleCounter = 0;
         $result = null;
+        $area = $this->areaList->getArea($this->appState->getAreaCode());
+        $area->load(Area::PART_DESIGN);
+        $area->load(Area::PART_TRANSLATE);
         while (!$request->isDispatched() && $routingCycleCounter++ < 100) {
```

Comment:
The only concern here is at the moment run into this method `dispatch` some reason what if area not available ?